src/utils/url.ts

```diff
@@ -36,6 +36,10 @@ const contentApiParameters = [
   'fullWidth',
 ]
 
+export const subjectStartPages: { [I in Instance]?: string[] } = {
+  de: ['/biologie', '/chemie', '/lerntipps', '/mathe', '/nachhaltigkeit'],
```

Comment:
Sorry, I don't understand this well enough to review yet.
- Why is informatik missing here?
- Why should those be handled differently from other "frontendSpecialPaths" like `/spenden` (see [here](https://github.com/serlo/serlo.org-cloudflare-worker/blob/main/src/frontend-proxy.ts#L138-L149))